### ISSUE TITLE
A few glitches

### ISSUE THREAD

**Issue by geonanorch** [JiHong88/SunEditor#7]
Thank you for suneditor, it is lightweight, works on mobiles and includes many features!

While integrating it I spotted a few glitches, here they are:
* format" button (2nd left):
  * class should not be `btn_size`
  * typo, should read `normal` instead of `nomal`
* initialization with `showTable: false` currently results in an exception

**Comment by JiHong88** [comment]
Thank you for having a good look at suneditor!
You've found the bug in fine detail :)
Fixed a bug you found

commit  : https://github.com/JiHong88/SunEditor/commit/27b6c6c3782a8fd2604603c3cf10129a2a995857

**Comment by geonanorch** [comment]
Thank you @JiHong88 . That is because I plan on making extensive use of your editor in my project :-)

I just retested, 2 out of 3 bugs are resolved: the typo (where under the `Format` dropdown the regular paragraph is labeled `nomal` instead of `normal`) seems to be still there, so leaving this issue open.

By the way I spent about 2 days evaluating open source Javascript WYSIWYG editors before choosing yours, I reckon you are offering a very good one. So thank you again!
